class ConflictError(Exception):
    pass

class RecordNotFound(Exception):
    pass